assessment/collectors/embeddings/generate_embedding.go

```diff
@@ -132,3 +101,46 @@ func createEmbededTextsWithClient(ctx context.Context, client PredictionClientIn
 	}
 	return concepts, nil
 }
+
+func createEmbededTextsWithClient(ctx context.Context, client PredictionClientInterface, project, location, model, sourceTargetFramework string) ([]MySqlMigrationConcept, error) {
```

Comment:
seems like this method was moved. If there is no other change, please move it back ?